### PR TITLE
Create task_annotation.rst doc

### PR DESCRIPTION
**Pull request by WangWinson**
This is add user guide to readdocs

@RackHD/corecommitters @cgx027 @iceiilin @pengz1

### CODE REVIEW

File: docs/rackhd/task_annotation.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+.. code-block:: shell
+
+    git clone https://github.com/RackHD/on-http
+    cd on-http
+    npm install
+    npm run taskdoc
+    
+
+You can access it via http://127.0.0.1/taskdoc, when on-http service is running
```
Comment (yyscamper) [inline docs/rackhd/task_annotation.rst]: Miss the port

File: docs/rackhd/task_annotation.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
+The RackHD Task Annotation is a web UI documentation for RackHD tasks.
+
+The UI page leverage the template of `apidoc <https://github.com/apidoc/apidoc>`_ to display the auto generated task_doc_data.json.
```
Comment (yyscamper) [inline docs/rackhd/task_annotation.rst]: I don't think it's good to write this in user guide, this is belongs to the design detail.

**Comment by benbp** [comment]
Should the _static .png file be included here?

File: docs/rackhd/task_annotation.rst (hunk 1 above)
Comment (benbp) [inline docs/rackhd/task_annotation.rst]: Probably better to say `via http(s)://<server>:<port>/taskdoc`

File: docs/rackhd/task_annotation.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+    git clone https://github.com/RackHD/on-http
+    cd on-http
+    npm install
+    npm run taskdoc
+    
+
+You can access it via http://127.0.0.1/taskdoc, when on-http service is running
+
+.. image:: /_static/task_annotation.png
```
Comment (benbp) [inline docs/rackhd/task_annotation.rst]: I'm kind of confused as to what this image is telling me. Mostly I see the node identify documentation, and a readthedocs link in an image.

File: docs/rackhd/task_annotation.rst (hunk 2 above)
Comment (benbp) [inline docs/rackhd/task_annotation.rst]: "The UI page leverages"

File: docs/rackhd/task_annotation.rst (hunk 3 above)
Comment (WangWinson) [inline docs/rackhd/task_annotation.rst]: Well, I try to put the screenshot example of the task annotation to show what it looks like

**Comment by iceiilin** [comment]
:+1:

File: docs/rackhd/task_annotation.rst (hunk 3 above)
Comment (WangWinson) [inline docs/rackhd/task_annotation.rst]: @benbp I just replace the sceenshot for install centos task. which is more representative.

File: docs/rackhd/task_annotation.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
+The RackHD Task Annotation is a web UI documentation for RackHD tasks.
+
+It mainly explains each option of every tasks according to the related task schema.
```
Comment (yyscamper) [inline docs/rackhd/task_annotation.rst]: every tasks --> every task

Comment (WangWinson) [inline docs/rackhd/task_annotation.rst]: Updated

**Comment by yyscamper** [comment]
:+1:

**Pull request merged**